File: Java/bubble_sort.java
import java.util.Arrays;

public class bubble_sort {
    public static void main(String[] args) {
        int[] arr = {12, 15, 13, 12, 47, 96};
        System.out.println(Arrays.toString(bubbleSort(arr)));
    }

    static int[] bubbleSort(int[] nums){
        int len = nums.length;
        if(len<=0){
            return nums;
        }else{
            for(int i=0; i<len; i++){
                for(int j=0; j<len-i-1; j++){
                    if(nums[j]>nums[j+1]){
                        int temp = nums[j];
                        nums[j] = nums[j+1];
                        nums[j+1] = temp;
                    }
                }
            }
        }
        return nums;
    }
}

File: Python/bubble_sort.py
def bubble_sort(nums):
    length = len(nums)
    if length <= 0:
        return nums
    else:
        for i in range(length):
            for j in range(length - i - 1):
                if nums[j] > nums[j + 1]:
                    nums[j], nums[j + 1] = nums[j + 1], nums[j]
    return nums


arr = [12, 15, 13, 12, 47, 96]
print(bubble_sort(arr))